packages/atoms/src/components/kiwi-modal/kiwi-modal.tsx

```diff
@@ -175,52 +101,22 @@ export class KiwiModal {
                   >
                     ×
                   </button>
+
                   <h4 class="modal-title">
-                    <slot name="modal-title"></slot>
+                    <slot name="kiwi-modal-title"></slot>
                   </h4>
                 </div>
               )}
+
               <div class="modal-body">
-                <slot name="modal-body"></slot>
+                <slot name="kiwi-modal-body"></slot>
               </div>
-              {some([
-                this.cancelText,
-                this.previousText,
-                this.nextText,
-                this.okText,
-              ]) && (
-                <div class="modal-footer">
-                  {this.previousText && (
```

Comment:
There is no replacement for this behaviour. Is it planned to include a `kiwi-modal-footer-stepped`?

Reply:
No it is not. I would argue that there actually was much behaviour here. It provides a button with a click handler and a label. Having dozens of properties for this, or creating dedicated components for each use case one can think of is not worth the effort in my opinion.

You can achieve the same result by using the `<kiwi-modal-footer>` with either custom labels, or completely custom markup. You might have to wright a little bit more markup with the latter, but you gain much more flexibility (like disabling buttons, loading buttons and the like) which with the "dedicated component" approach could only be achieved by introducing more complexity, which I think is not worth it.